src/vs/workbench/services/runtimeStartup/common/runtimeStartup.ts

```diff
@@ -182,6 +186,16 @@ export class RuntimeStartupService extends Disposable implements IRuntimeStartup
 			}
 		}));
 
+		// Add the onDidCompleteDiscoveryPhase event handler.
+		this._register(this.onDidCompleteDiscoveryPhase(() => {
+			if (this._languageRuntimeService.registeredRuntimes.length === 0) {
+				// if no runtimes were found, notify the user about the problem
+				this._notificationService.error(nls.localize('noRuntimesMessage',
+					"No runtimes found. Please see the [Get Started](https://positron.posit.co/start.html) \
+					page for instructions on how to add a runtime."));
```

Comment:
I don't consider myself the best wordsmith so if there's any suggestions on improving this message please let me know!